pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -430,6 +502,40 @@ func (c *Cluster) getCopyBinariesContainer() (v1.Volume, *v1.Container) {
 	}
 }
 
+// This container runs all the actions needed to activate an OSD before we can run the OSD process
+func (c *Cluster) getActivateOSDInitContainer(volumeMounts []v1.VolumeMount, osdID, osdUUID string, isFilestore bool, resources v1.ResourceRequirements) (v1.Volume, *v1.Container) {
+	volume := v1.Volume{Name: activateOSDVolumeName, VolumeSource: v1.VolumeSource{EmptyDir: &v1.EmptyDirVolumeSource{}}}
+	envVars := cephVolumeEnvVar()
+	osdStore := "--bluestore"
+	if isFilestore {
+		osdStore = "--filestore"
+	}
+
+	volMounts := []v1.VolumeMount{
+		{Name: activateOSDVolumeName, MountPath: activateOSDMountPath},
+		{Name: "devices", MountPath: "/dev"},
```

Comment:
Can this activate an OSD without needing all of `/dev`? What if it just had `/dev/<disk>`?